README.md

```diff
@@ -15,6 +15,30 @@ Join our chat on the [bids-matlab channel](https://mattermost.brainhack.org/brai
 See also [PyBIDS](https://github.com/bids-standard/pybids) for Python and the [BIDS Starter Kit](https://github.com/bids-standard/bids-starter-kit).
 
 
+## Features
+
+### What this toolbox can do
+
+- read the layout of a BIDS dataset (see `bids.layout`),
+- perform queries on that layout to get information about the subjects, sessions, runs, modalities, metadata... contained within that dataset (see `bids.query`),
+- generate a human readable report of the content of BIDS data set containing anatomical MRI, functional MRI, diffusion weighted imaging, field map data (see `bids.report`)
+- read and write JSON files (see `bids.util.jsondecode` and `bids.util.jsonencode`) provided that the right [dependencies](#reading-and-writing-json-files) are installed,
+- read and write TSV files (see `bids.util.tsvread` and `bids.util.tsvwrite`),
+
+The behavior of this toolbox assumes that it is interacting with a valid BIDS dataset that should have been validated using [BIDS-validator](https://bids-standard.github.io/bids-validator/). Any unvalidated components may produce undefined behavior. Although, if you're BIDS-y enough, the behavior may be predictable.
+
+
+### What this toolbox cannot do... yet
+
+- parse the layout of "BIDS-derivative compatible" datasets (like fMRIprep),
+- create BIDS compatible filenames or folder structures for raw or derivatives datasets,
+- basic copying of files to help with derivative dataset generation,
```

Comment:
```suggestion
- do basic copying of files to help with derivative dataset generation,
```
This should probably be a verb phrase, to satisfy parallel construction of the list items.